jenkins-integration/beaker/install/shared/stop_sut_background_scripts.rb

```diff
@@ -0,0 +1,12 @@
+require 'json'
+
+step "Stop background scripts on SUT" do
+  pids = JSON.parse(File.read("bg_pids.json"))
+
+  Beaker::Log.notify("Stopping #{pids.count} background scripts")
+  Beaker::Log.notify(JSON.pretty_generate(pids))
+  pids.each do |k, v|
+    Beaker::Log.notify("Stopping script '#{k}' with pid '#{v}'")
+    on(master, "kill #{v}")
```

Comment:
Should we do some kind of a wait after each `kill` command to validate that the process is dead before moving on?  Maybe something like:

```ruby
on (master, "while kill -0 #{v} 2>/dev/null; do sleep 0.5; done")
```